src/language/runtime/environment.ts

```diff
@@ -469,3 +637,10 @@ export class ClassPrototype {
         return land(undefined);
     }
 }
+
+
+// General internal class declarations
+export interface ProxyObjectVal<C> extends ObjectVal {
+    o: C;
```

Comment:
An sich kein so guter Name. Die ein-buchstabige Konvention bei Robot und World war ein Fehler xd. Vorschlag

```suggestion
// General internal class declarations
export interface NativeObjectVal<C> extends ObjectVal {
    nativeRepr: C;
}
```

Reply:
Ja, gefällt mir besser 👍